release-tools/prow.sh

```diff
@@ -652,103 +515,74 @@ find_deployment () {
+  kubectl apply -f "${CRD_BASE_DIR}/objectstorage.k8s.io_bucketaccesses.yaml" --validate=false
   cnt=0
-  until kubectl get volumesnapshotclasses.snapshot.storage.k8s.io \
-    && kubectl get volumesnapshots.snapshot.storage.k8s.io \
-    && kubectl get volumesnapshotcontents.snapshot.storage.k8s.io; do
+  until kubectl get bucketaccessclasses.objectstorage.k8s.io \
+    && kubectl get bucketaccessrequests.objectstorage.k8s.io \
+    && kubectl get bucketaccesses.objectstorage.k8s.io \
+    && kubectl get bucketclasses.objectstorage.k8s.io	 \
+    && kubectl get bucketrequests.objectstorage.k8s.io  \
+    && kubectl get buckets.objectstorage.k8s.io; do
     if [ $cnt -gt 30 ]; then
-        echo >&2 "ERROR: snapshot CRDs not ready after over 1 min"
+        echo >&2 "ERROR: cosi CRDs not ready after over 1 min"
         exit 1
     fi
-    echo "$(date +%H:%M:%S)" "waiting for snapshot CRDs, attempt #$cnt"
+    echo "$(date +%H:%M:%S)" "waiting for cosi CRDs, attempt #$cnt"
 	cnt=$((cnt + 1))
     sleep 2
   done
 }
 
-# Install snapshot controller and associated RBAC, retrying until the pod is running.
-install_snapshot_controller() {
-  kubectl apply -f "https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${CSI_SNAPSHOTTER_VERSION}/deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml"
+# Install controller and associated RBAC, retrying until the pod is running.
+install_controller() {
+  #kubectl apply -f "https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-controller/${COSI_CONTROLLER_VERSION}/deploy/controller.yaml"
+  kubectl apply -f "https://raw.githubusercontent.com/container-object-storage-interface/cosi-controller-manager/master/samples/rbac.yaml"
   cnt=0
-  until kubectl get clusterrolebinding snapshot-controller-role; do
+  until kubectl get clusterrolebinding cosi:system; do
      if [ $cnt -gt 30 ]; then
         echo "Cluster role bindings:"
         kubectl describe clusterrolebinding
-        echo >&2 "ERROR: snapshot controller RBAC not ready after over 5 min"
+        echo >&2 "ERROR: controller RBAC not ready after over 5 min"
         exit 1
-    fi
-    echo "$(date +%H:%M:%S)" "waiting for snapshot RBAC setup complete, attempt #$cnt"
-	cnt=$((cnt + 1))
-    sleep 10   
+     fi
+     echo "$(date +%H:%M:%S)" "waiting for cosi RBAC setup complete, attempt #$cnt"
+	 cnt=$((cnt + 1))
+     sleep 10   
   done
 
 
-  kubectl apply -f "https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${CSI_SNAPSHOTTER_VERSION}/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml"
+  #kubectl apply -f "https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-controller/${COSI_CONTROLLER_VERSION}/deploy/rbac.yaml"
+  kubectl apply -f "https://raw.githubusercontent.com/brahmaroutu/cosi-controller-manager/test/samples/controller.yaml"
```

Comment:
See my comments earlier about install location should be dynamic.

Reply:
Fixed